src/intervals/rounding.jl

```diff
@@ -9,6 +9,7 @@ interval rounding types, e.g.
 +(IntervalRounding{:none}, a, b, RoundDown)
 
 The current allowed rounding types are
+- :errorfree # use errorfree arithmetic via FastRounding.jl
```

Comment:
I shall still refer to `:errorfree` instead of (the new) `:fast` to avoid confusions.